omnibus/config/projects/push-jobs-client.rb

```diff
@@ -24,7 +24,7 @@
 conflict "opscode-push-jobs-client"
 
 build_iteration 1
-build_version "2.0.2"
+build_version "2.1.0"
```

Comment:
@ksubrama convinced me about this change but I'm hazy on the details so he's welcome to chime in. From what I remember a minor bump seemed appropriate given recent changes in omnibus-software.